R/rank.R

```diff
@@ -36,9 +33,15 @@
 #'   non-duplicate value will have a rank incremented by `1`, effectively
 #'   removing any gaps in the ranking.
 #'
-#' @param na_propagate A single logical specifying whether or not missing
-#'   values should be propagated. If `TRUE`, all missing values are given
-#'   the rank `NA`.
+#' @param incomplete Treatment of [incomplete][vec_detect_complete]
```

Comment:
Maybe:

> Treatment of missing observations (in the [incomplete][vec_detect_complete] sense).

And then use missing in the rest of the text? I think it's hard to reason about the doc with the term incomplete. Especially with non-rows values. I would not have first guessed I should use this argument for `NA` values. Of course they are incomplete, but in my mind they are missing and completeness is a property I associate with rows.

Maybe we should hold this renaming? Something doesn't feel right. I'd be tempted to rename to `missing` instead of `incomplete`, and then use documentation to disambiguate.

Reply:
Okay we can hold off on merging this immediately, I don't have a perfect solution yet either

Note that `na_value` also says:

> Treatment of missing values.